crates/cli-support/src/wasm2es6js.rs

```diff
@@ -63,7 +63,11 @@ fn args_are_optional(name: &str) -> bool {
 pub fn interface(module: &Module) -> Result<String, Error> {
     let mut exports = String::new();
     module_export_types(module, |name, ty| {
-        writeln!(exports, "  readonly {name}: {ty};").unwrap();
+        if name.contains(':') {
+            writeln!(exports, "  readonly {name:?}: {ty};").unwrap();
```

Comment:
Its not entirely clear on first glance whats going on here. I think a comment would help.